src/dataset.ts

```diff
@@ -317,15 +348,27 @@ export class Savvy implements DataSet {
         ]);
     }
     /**
-     * A map of of unique column keys to longer descriptions
+     * A map of of unique column keys to scale levels and their labels
+     */
+    public get levels() : Map<string, Map<number, string>> {
+        return(new Map([...this._levels]));
+    }
+    public set levels(levels : Map<string, Map<number, string>>) {
```

Comment:
We probably want to do a validation here, such that the incoming levels match up with the actual levels in the column. This is sort of the benefit of a set method, but we haven't used it so far because the other setters are pretty simple. But we could do the same there. Not necessary for this PR though.